src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -323,7 +323,12 @@ public void run()
             }
           }
           else {
-            register(channel, SelectionKey.OP_CONNECT, l);
+            /*
```

Comment:
he has another solution where if the select returns w/o timeout but returns zero descriptors, he is able to identify the ones which are connected. I wonder if we should be doing that as well.